measures/ResStockArguments/resources/electrical_panel.rb

```diff
@@ -179,4 +253,69 @@ def simplify_fuel_type(fuel_type)
       return 'non-electricity'
     end
   end
+
+  def get_major_elec_load_count(args)
+    # has electric primary heating
+    has_elec_heating_primary = electric_fuel(args[:heating_system_fuel])
+    # emulate HVAC Heating Type 
+    hvac_heating_type = convert_heating_type(args[:heat_pump_type])
+
+    # has electric water heater
+    has_elec_water_heater = electric_fuel(args[:water_heater_fuel_type])
+
+    # emulate HVAC Cooling Type 
+    hvac_cooling_type = convert_cooling_type(args[:cooling_system_type], args[:heat_pump_type])
+    has_cooling = has_cooling(hvac_cooling_type)
+
+    # appliance presence and electric fuel
+    has_elec_drying = electric_fuel_and_presence(args[:clothes_dryer_present], args[:clothes_dryer_fuel_type])
+    has_elec_cooking = electric_fuel_and_presence(args[:cooking_range_oven_present], args[:cooking_range_fuel_type])
+    has_pv = has_pv(args[:pv_system_present])
+    has_ev_charging = 0
+
+    load_vars = [
+      has_elec_heating_primary,
+      has_elec_water_heater,
+      has_elec_drying,
+      has_elec_cooking,
+      has_cooling,
+      has_ev_charging,
+      has_pv
+    ]
+    load_count = load_vars.sum
+
+    #Count heat pump only once if it provides heating and cooling
+    if hvac_cooling_type == "heat pump" and hvac_heating_type == "ASHP or GHP"
+      load_count = load_count - 1 #Assuming a single heat pump provides heating and cooling
+    #Don't count plug-in Room ACs as major loads
+    elsif hvac_cooling_type == "room air conditioner"
```

Comment:
I think there's a room AC constant you can use here: https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/hpxml.rb